### ISSUE TITLE
iron-a11y-announcer doesn't announce all messages on mac

### ISSUE THREAD

**Issue by nik3daz** [PolymerElements/iron-a11y-announcer#33]
If two messages are announced in quick succession on mac, voiceover only reads the second message. iron-a11y-announcer should probably queue messages and update them at reasonable times so that no messages get dropped.

**Comment by TomHennen** [comment]
I've also been experiencing messages not being announced.
